nistats/permutation_tests.py

```diff
@@ -0,0 +1,296 @@
+"""
+Permutation tests for first level and second level analysis.
+"""
+# Author: Martin Perez-Guevara, <mperezguevara@gmail.com>, jan. 2016
+import warnings
+import numpy as np
+from sklearn.utils import check_random_state
+import sklearn.externals.joblib as joblib
+from sklearn.base import clone
+from nistats.design_matrix import make_design_matrix
+import time
+import sys
+
+
+def _meet_contrast_conditions(con_val):
+    """Check if permutation test is allowed for given contrast values."""
+    # No F tests
+    if type(con_val[0]) is list:
```

Comment:
Yes, this should checked by other means. 
I think that the API would clearer if you were providing the identifiers (strings) of two columns of the design matrix on the difference of which you want to make inference.